contrib/upgrade_python-mastodon.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# upgrade the python-mastodon module if Debian/RPM package was outdated
+# -----------------------------------------------------------------------------
+# this is intended for the case your distribution ships with an outdated version
+# of the python-mastodon package (pre-1.5.1). It will simply replace the module
+# scripts where your packaging placed them.
+#
+# © 2022 Izzy <izzysoft AT qumran DOT org>; GPL-3.0-or-later
+
+# -----------------------------------------------------------------------------
+# define temp dir and exit-on-error
+function cleanup() {
+  echo
+  echo "Something went wrong, aborting."
+  rm -rf $tmpdir
+  exit 1
+}
+
+tmpdir=$(mktemp -d)
+trap cleanup ERR
+
+
+# ==============
+# ===[ Main ]===
+# ==============
+# -----------------------------------------------------------------------------
+# check whether we need to update python3-mastodon
+echo
+echo "Let's see if your distribution had the recent version of python3-mastodon…"
```

Comment:
"Let's see if your distribution has a recent version of python3-mastodon…"

Regarding the use of non-ASCII in print strings, what's the consensus on that? I mean, it's how I like to do things, but it it's still tricky.

Reply:
You mean the ellipsis? Afraid that was just "muscle memory" from writing articles the past few days. Agreed, I'll make them simple dots then.